homeassistant/components/wled/const.py

```diff
@@ -17,6 +17,7 @@
 ATTR_PALETTE = "palette"
 ATTR_PLAYLIST = "playlist"
 ATTR_PRESET = "preset"
+ATTR_REVERSE = "reverse"
```

Comment:
should this be called `effect_reverse` to make it clear what is being reversed ?

Reply:
The same can be said about `speed` for example.
I kept it consistent with that and the way it is used/displayed in WLED itself, just to stay close with that.